fastify.js

```diff
@@ -219,6 +219,9 @@ function fastify (options) {
     // custom parsers
     addContentTypeParser: ContentTypeParser.helpers.addContentTypeParser,
     hasContentTypeParser: ContentTypeParser.helpers.hasContentTypeParser,
+    defaultJSONParser: ContentTypeParser.defaultParsers.getDefaultJsonParser((options.onProtoPoisoning || defaultInitOptions.onProtoPoisoning),
```

Comment:
I think we are duplicating the creation of this parser. Can we reuse the other one?

Reply:
Did you mean this part is duplicated `(options.onProtoPoisoning || defaultInitOptions.onProtoPoisoning)`

Reply:
Oops got it ! Thanks